Formula/prototool.rb

```diff
@@ -0,0 +1,31 @@
+class Prototool < Formula
+  desc "Your Swiss Army Knife for Protocol Buffers"
+  homepage "https://github.com/uber/prototool"
+  url "https://github.com/uber/prototool/archive/v1.1.0.tar.gz"
+  sha256 "7759bd4db9a907d3dea4abb34ff18ddbc814f4de56d43aca8d2f402e6fd623fa"
+
+  depends_on "automake" => :build
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    prototoolpath = buildpath/"src/github.com/uber/prototool"
```

Comment:
Use `dir` instead of `prototoolpath` please, for the sake of trying to maintain consistency.

Reply:
Updating.